src/filtrering/filtrering-filter/filterform/aktivitet-filterform.tsx

```diff
@@ -0,0 +1,122 @@
+            ...valgteAktiviteter,
+            [aktivitetKey]: verdi
+        });
+    };
+
+    const harValg =
+        valgteAktiviteter.BEHANDLING !== 'NA' ||
+        valgteAktiviteter.EGEN !== 'NA' ||
+        valgteAktiviteter.GRUPPEAKTIVITET !== 'NA' ||
+        valgteAktiviteter.IJOBB !== 'NA' ||
+        valgteAktiviteter.MOTE !== 'NA' ||
+        valgteAktiviteter.SOKEAVTALE !== 'NA' ||
+        valgteAktiviteter.STILLING !== 'NA' ||
+        valgteAktiviteter.TILTAK !== 'NA' ||
+        valgteAktiviteter.UTDANNINGAKTIVITET !== 'NA';
+
+    const nullstillAktiviteter = () => {
+        setValgteAktiviteter(aktivitetInitialState);
+        endreFiltervalg('aktiviteter', aktivitetInitialState);
+    };
+
+    return (
+        <form className="skjema aktivitetfilterform" data-testid="aktivitet-filterform">
+            <div className="aktivitetvalg__header blokk-xxs">
+                <span className="aktivitetvalg__header--first">Ja</span>
+                <span>Nei</span>
+            </div>
+            <div className="aktivitetfilterform__valg">
+                {Object.entries(valg).map(([kode, verdi]) => [
+                    <div
+                        key={`skjemaelement skjemaelement--horisontal aktivitet-${kode}`}
+                        className="aktivitetvalg blokk-xxs"
+                    >
+                        <span className="aktivitetvalg__tekst">{verdi as string}</span>
+                        <div className="radioknapp-gruppe">
+                            <input
+                                id={`aktivitet-${kode}-ja`}
+                                name={kode}
+                                value="JA"
+                                type="radio"
+                                checked={valgteAktiviteter[kode] === 'JA'}
+                                className="skjemaelement__input radioknapp"
+                                onChange={() => handleRadioChange(kode, 'JA')}
+                                key={`Ja, ${verdi}`}
+                                data-testid={`aktivitet-filterform-${kode}-ja`}
+                            />
+                            <label
+                                htmlFor={`aktivitet-${kode}-ja`}
+                                className="skjemaelement__label aktivitet_radioknapp_label"
+                            >
+                                <span className="sr-only">Ja, {verdi}</span>
+                            </label>
+                            <input
+                                id={`aktivitet-${kode}-nei`}
+                                name={kode}
+                                value="NEI"
+                                type="radio"
+                                checked={valgteAktiviteter[kode] === 'NEI'}
+                                className="skjemaelement__input radioknapp"
+                                onChange={() => handleRadioChange(kode, 'NEI')}
+                                key={`NEJ, ${verdi}`}
```

Comment:
can you pls check Is it _NEJ_ in key correct

Reply:
I changed it to NEI now, I think it still works?

Reply:
Yep, I am not sure is it used at all....it still works.